packages/paste-core/primitives/text/stories/index.stories.tsx

```diff
@@ -102,9 +102,22 @@ storiesOf('Primitives|Text', module)
         fontSize={['fontSize40', 'fontSize50', 'fontSize60', 'fontSize70']}
         padding={['space0', 'space10', 'space130', 'space70']}
         textAlign={['center', 'right', 'left']}
-        textColor={['colorTextBrandHighlight', 'colorTextLink', 'colorTextSuccess', 'colorText']}
+        color={['colorTextBrandHighlight', 'colorTextLink', 'colorTextSuccess', 'colorText']}
       >
         Some text with responsive styling.
       </Text>
     );
+  })
+  .add('Pseudo-classes with props', () => {
```

Comment:
I wouldn't mind an additional test or 2 in addition to the story. Just to prove that the pseudo selectors are working and outputting css we expect.

Reply:
I've added a few more stories. Let me know if those will work.

Reply:
Oh, I meant some jest tests, maybe snapshots. Something that proves the code produces the css you expect

Reply:
Oh I see that now. I read that wrong. I removed those stories and added some snapshot tests.

Reply:
Updated tests to test all pseudo-classes.